### ISSUE TITLE
"permission denied" when executing a step inside centos7 container

### ISSUE THREAD

**Issue by chrispine** [sourcegraph/sourcegraph#15284]
- **Sourcegraph version:** 3.19.2 (and various version of src-cli)
- **Platform information:** `centos7` Docker image

From [this Slack thread](https://sourcegraph.slack.com/archives/CMMTWQQ49/p1604342875063100), user reports getting "permission denied" when executing a step inside a `centos7` Docker container. While customer was not on the latest version, it didn't seem that that was the cause of this issue.

**Comment by mrnugget** [comment]
Probably needs to be mentioned that it wasn't the standard `centos:7` image, but a custom one and the command they wanted to run was this:

```
docker run <redacted>docker-build-environment-base-centos7:latest-stable "gradle lockdown --update <redacted>"
```

**Comment by chrispine** [comment]
@LawnGnome IIRC, you fixed this already. If so, go ahead and close this ticket.

**Comment by LawnGnome** [comment]
Fixed in sourcegraph/src-cli#366.
